docs/photoniq/event-delivery/getting-started-event-delivery.md

```diff
@@ -0,0 +1,196 @@
+---
+sidebar_position: 10
+title: Getting Started with Event Delivery
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+PhotonIQ Event Delivery is a fully managed service, implemented and optimized by Macrometa engineers to ensure proper functionality. For those who prefer a more hands-on approach, this guide provides information to help you actively manage Event Delivery.
+
+In this guide, you'll learn how to:
+- [Set up your first event stream in the PhotonIQ GDN](#setup-event-stream-in-the-photoniq-gdn)
+- [Subscribe to events and recieve real time updates](#subscribe-to-events-and-recieve-real-time-updates)
+- [Monitor the Event Delivery metrics and health stautus](#monitor-the-event-delivery-metrics-and-health-stautus)
+
+
+The Event Delivery service enables you to stream and filter events from Macrometa [streams](../../streams/), as well as publish events to these streams. Before you proceed, here are some key concepts:
+
+- **Publisher:** An application or service that continuously generates data for consumption by a subscriber. In the Event Delivery Service (EDS), collections in the Macrometa Global Data Network (GDN) act as publishers.
+  
+- **Subscriber:** An entity that subscribes to events from the publisher. Subscribers can use SQL-like queries to filter and subscribe to specific events, reducing noise and focusing on relevant data.
+  
+- **Stream:** A named channel for sending messages. Every stream in the GDN is backed by a distributed append-only log and can be either local (restricted to one edge location) or global (spanning all edge locations in the [fabric](../../geofabrics/)).
+  
+- **Event:** Each new data point or message generated by the publisher.
+
+
+## Setup event stream in the PhotonIQ GDN
+
+**Prerequisites**
+- A Macrometa account with sufficient permissions to create [collections](../../collections/). You can contact a Macrometa partner to sign up or [sign up](https://www.macrometa.com/sign-up) on your own. Once your Macrometa account is set up, you can choose the type of collection to stream your data into. The PhotonIQ GDN offers the following options for EDS:
```

Comment:
As part of account setup, there will need to be 2 explicit steps. 1. GDN account setup and 2. EDS config update with the customer information  like customer id, fabric etc. Maybe this will not be required to mention in the user facing doc, but we could mention that MM will provide the EDS customer info.